lib/kernel-manager.js

```diff
@@ -86,11 +86,26 @@ class KernelManager {
 
   startKernel(kernelSpec, grammar, onStarted) {
     const displayName = kernelSpec.display_name;
+    let kernelStartDir;
+    let currentPath = getEditorDirectory(store.editor);
 
     log("KernelManager: startKernel:", displayName);
 
+    switch (atom.config.get("Hydrogen.startDir")) {
+      case "firstProjectDir":
+        if (atom.project.getPaths().length > 0) {
+          kernelStartDir = atom.project.getPaths()[0];
+          break;
+        }
+      case "projectDirOfFile":
+        kernelStartDir = atom.project.relativizePath(currentPath)[0];
```

Comment:
Was just going to write and say we need to catch this :) According to the docs it should return `null` if there is no projectpath? Need to test this. 

Is the way to test "without" a project path to open a file directly in atom?